### PR TITLE
Subgraph queries for UI and some refactoring

### PR DESCRIPTION
**Pull request by 0xZenus**

### CODE REVIEW

File: src/entities/amm/voltz-config/config.ts [hunk 1]
```diff
@@ -0,0 +1,244 @@
+import { NetworkConfiguration } from './types';
```
Comment (0xZenus) [inline src/entities/amm/voltz-config/config.ts:1]: Copied from UI.

File: src/entities/amm/voltz-config/getConfig.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { providers } from 'ethers';
```
Comment (0xZenus) [inline src/entities/amm/voltz-config/getConfig.ts:1]: Copied from UI.

File: src/entities/amm/voltz-config/types.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+type PoolConfiguration = {
```
Comment (0xZenus) [inline src/entities/amm/voltz-config/types.ts:1]: Copied from UI.

File: src/entities/amm/amm.ts [hunk 4]
```diff
@@ -85,56 +83,50 @@ export class AMM {
...
+    this.termEndTimestampInMS = termEndTimestampInMS;
     this.underlyingToken = underlyingToken;
     this.tickSpacing = tickSpacing;
     this.wethAddress = wethAddress;
     this.isETH = this.underlyingToken.name === 'ETH';
     this.ethPrice =
       ethPrice || (() => geckoEthToUsd(process.env.REACT_APP_COINGECKO_API_KEY || ''));
+
+    const chosenProvider = provider || signer?.provider;
+    if (!chosenProvider) {
```
Comment (ioanaroceanu2000) [inline src/entities/amm/amm.ts:108]: provider is no longer an optional param, right? do we still need this check?

File: src/entities/amm/voltz-config/config.ts (hunk 1 above)
Comment (ioanaroceanu2000) [reply src/entities/amm/voltz-config/config.ts:1]: What's this used for?

File: src/init.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import {
+  BrowserClient,
+  makeFetchTransport,
+  defaultStackParser,
+  defaultIntegrations,
+} from '@sentry/browser';
+
+let sentryTracker: BrowserClient | null = null;
+
+export const init = (): void => {
```
Comment (ioanaroceanu2000) [inline src/init.ts:10]: is the `init` name a convention? I'd change it to something like `sentryInit`

File: src/entities/amm/voltz-config/config.ts (hunk 1 above)
Comment (0xZenus) [reply src/entities/amm/voltz-config/config.ts:1]: To the whitelists and so. It's on the UI rn but we'd like to move it.

File: src/init.ts (hunk 5 above)
Comment (0xZenus) [reply src/init.ts:10]: hmm, I agree but we could append more to this ``init`` function.

**Review by ioanaroceanu2000 — APPROVED** [review]

**Pull request merged**